python/tvm/script/printer/doc.py

```diff
@@ -29,12 +34,63 @@ class Doc(Object):
 class ExprDoc(Object):
     """Base class of all expression Docs"""
 
+    def attr_access(self, attr: str) -> "AttrAccessDoc":
```

Comment:
Let's be consist with C++ APIs

```suggestion
    def attr(self, attr: str) -> "AttrAccessDoc":
```

Reply:
This is a tricky one. If strictly following C++ APIs and Python name convention, we will have
```python
doc.attr  # helper method to create AttrAccessDoc
doc.attr  # AttrAccessDoc's `attr` attribute, that's a name conflict

doc.index  # helper method to create IndexDoc
doc.indices  # attribute of IndexDoc, looks similar to the method above
```

An option is to update the C++ API to `AttrAccess` and `IndexAccess`, which makes the code slightly more verbose
```cpp
TIR(p)->Attr("cast")
// becomes
TIR(p)->AttrAccess("cast")
```

What do you think?

Reply:
Sounds good, updated according to your recommendation